### ISSUE TITLE
Use of the internal package is not allowed

### ISSUE THREAD

**Issue by 1486756632** [edgexfoundry/device-modbus-go#300]
##
![808b4b03d46cc1877f9b96c60d7b5fb](https://user-images.githubusercontent.com/51595387/143859099-52fb5c54-d7d6-4832-9c14-3d570b22be68.png)
# 
i use goland to compile code. Why there is always a error "Use of the internal package is not allowed". I pull the code from github. I do not change any code. Is there anyone could help me.Thanks a lot

**Comment by cloudxxx8** [comment]
that's the design of golang.  you can't use the internal package of other repository.
https://go.dev/doc/go1.4#internalpackages

**Comment by 1486756632** [comment]
> that's the design of golang. you can't use the internal package of other repository. https://go.dev/doc/go1.4#internalpackages

There is a question.How can run the code ?How can i debug the code?

**Comment by cloudxxx8** [comment]
no, you can't use the internal package from another repository.

**Comment by 1486756632** [comment]
> repository

Actually, i wanna rebuild the code.Would u mind tell me ,if i rebuild the code .How could i run main.go by GoLand.

**Comment by cloudxxx8** [comment]
from the screenshot, you are developing your code from another repository.
you have to clone this repository and put your main.go into it, and it will work.

**Comment by lenny-intel** [comment]
@1486756632 , can this issue be closed?
